### PR TITLE
refactor: authContext, add: user-profile

### PR DESCRIPTION
**Pull request by BelideNikhil**
+ User Profile Page

image

### CODE REVIEW

**Review by Rohitprasad83 — APPROVED** [review]
Code looks good and readable.

File: src/utils.css [hunk 1]
```diff
@@ -21,6 +21,23 @@
 .child-wrapper {
     flex-grow: 1;
 }
+/*  */
```
Comment (manojasarsa) [inline src/utils.css:24]: You can remove this comment

**Review by manojasarsa — APPROVED** [review]
User Profile looks good.
Just one sugg: If you can give some space between user profile box and footer. It will look more clean.

**Review by KoushikKintali — APPROVED** [review]
Everything looks clean, good to merge

**Pull request merged**